beetle/iroh-bitswap/src/lib.rs

```diff
@@ -417,29 +426,23 @@ impl<S: Store> NetworkBehaviour for Bitswap<S> {
             libp2p::swarm::FromSwarm::ConnectionClosed(event) => {
                 self.pause_dialing = false;
                 if event.remaining_established == 0 {
-                    // Last connection, close it
```

Comment:
I _think_ "last" actually was correct here since it's checking if `remaining_established=0`.